validation-test/StdlibUnittest/InterpreterHandlesExpectCrash.swift

```diff
@@ -0,0 +1,15 @@
+// RUN: %target-run-simple-swift -interpret
```

Comment:
This substitution is not designed to accept parameters to the compiler.  I'm not sure it even works the way you expect on OS X, but it definitely won't work on iOS.

I think the existing StdlibUnittest tests are sufficient, as long as they are run with the interpreter.